wiki/fr/fr-Installation-for-Macintosh.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: wiki
+title: "Installation sous MacOS"
+lang: "fr"
+permalink: "/wiki/Installation-for-Macintosh"
+---
+
+# Installation sous MacOS
+
+Assurez-vous de lire le [guide de démarrage et installation](Getting-Started) avant de commencer.
+
+1. [Téléchargez Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
+1. **Décompressez le fichier .zip téléchargé**. Double-cliquez sur le fichier `.zip`, ce qui créera un nouveau dossier du même nom. Le dossier contient le fichier de licence et un fichier `.dmg` contenant l'application.
+1. **Double-cliquez sur le fichier `.dmg`** pour l'ouvrir. L'image s'ouvre et vous verrez deux applications (client et serveur Jamulus).
+1. **Glissez et déposez les deux icônes dans le dossier Applications** pour installer Jamulus.
+1. **Executez Jamulus**. Vous devriez maintenant pouvoir utiliser Jamulus comme n'importe quelle autre application. Si vous voyez un message qui vous dit que Jamulus ne peut pas être ouvert parce que le développeur ne peut pas être vérifié, lisez la suite et jetez un coup d'oeil à la [section suivante](#jamulus-ne-peut-pas-être-ouvert-car-le-développeur-ne-peut-pas-être-vérifié).
+
+_Vous pouvez supprimer le dossier dans le répertoire Downloads contenant le `.dmg` et éjecter le lecteur "Jamulus" de votre bureau. Ils ne sont plus nécessaires._
+
```

Comment:
I'm not a Mac user and I wonder 2 things: 
1. _le répertoire Downloads_:  if the user is using a French locale, won't that be the "_Téléchargements_" folder?
2. _le lecteur "Jamulus"_: is the proper name really a "_lecteur Jamulus_" on a French Mac?

If you have a Mac, please double-check. If not, we'll need to find someone to check that for us.

Reply:
Yes in the GUI where it is localized, not in the Terminal (`bash`or `zsh` now) where it is `~/Downloads`.
I'll change here, we are in the GUI.

Reply:
On on the French MacOS UI, that is « dossier », not « répertoire ». I'll change that too.

Reply:
@gegeweb : it seems that you forgot tp answer to my `2. is the proper name really a "lecteur Jamulus" on a French Mac?` Can you please do so then we are sure we match what the user will actually experience?

Reply:
@trebmuh I used « lecteur » as shortcut for « lecteur disque », changed to « volume » because image disk as other disk are mounted in `/Volumes`. Could use `image` too.

Reply:
I'm not qualified with MacOs terminology so I'm leaving it to you.

Reply:
Since Catalina (10.15) MacOS is used instead of Mac OS X.

In fact, should be macOS not MacOS.

<img width="489" alt="Capture d’écran 2020-11-26 à 22 07 41" src="https://user-images.githubusercontent.com/9108457/100391472-d29e7f00-3033-11eb-8e96-fea522bbd2be.png">